front/client/components/ArticleContent.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <div>
+    <div v-for="(value, index) in contentItem" :key="index"
```

Comment:
If there are more than one attribute move each to its own line like this: 
`<div
   v-for="blablabla"
   :key="blalbal"
 /> 
`